src/Compilers/VisualBasic/Portable/Parser/Parser.vb

```diff
@@ -6093,6 +6093,13 @@ checkNullable:
             End If
         End Sub
 
+        Friend Shared Function CheckFeatures(feature As Feature, opts As VisualBasicParseOptions) As Boolean
```

Comment:
Maybe that one in the `scanner` shouldn't be there and with these in the `scanner`.